source/manual/replicate-app-data-locally.html.md

```diff
@@ -34,57 +34,72 @@ To get production data on to your local VM, you'll need to have either:
-1. Create an access token
+### Create an access token
 
-    Under `Access keys` click `Create access key` – you will secret is only displayed once, however if you fail to note it down just remove it and create another.
+Under `Access keys` click `Create access key` – you will secret is only displayed once, however if you fail to note it down just remove it and create another.
 
-1. Install the AWS CLI application locally
+### Install the AWS CLI application locally
 
-    On macOS this can be done using `brew install awscli`. This is being installed locally with the assumption you will be doing a two step backup process, i.e. download the files on your local machine and then update you VM. This is the recommended process as the download is quicker to you local machine for most users.
+On macOS this can be done using `brew install awscli`. This is being installed locally with the assumption you will be doing a two step backup process, i.e. download the files on your local machine and then update you VM. This is the recommended process as the download is quicker to you local machine for most users.
 
-1. Setup your AWS access config and credentials files
+### Setup your AWS access config and credentials files
 
-    I am repeating the instructions described in the [AWS setup guide](/manual/user-management-in-aws.html) with one difference. In the credentials file the setting must be under default as otherwise they are not found by the replication script.
+I am repeating the instructions described in the [AWS setup guide](/manual/user-management-in-aws.html) with one difference. In the credentials file the setting must be under default as otherwise they are not found by the replication script.
 
-    Create a `~/.aws/config` file:
+Create a `~/.aws/config` file:
+
+```
+[profile govuk-integration]
+role_arn = <Role ARN>
+mfa_serial = <MFA ARN>
+source_profile = gds
+region = eu-west-1
+```
+
+#### Role ARN
+Should be one of the following depending on the level of access you have been configured with [here](https://github.com/alphagov/govuk-aws-data/blob/master/data/infra-security/integration/common.tfvars):
+
+  * govuk-admins
 
     ```
-    [profile govuk-integration]
-    role_arn = <Role ARN>
-    mfa_serial = <MFA ARN>
-    source_profile = gds
-    region = eu-west-1
+    arn:aws:iam::210287912431:role/govuk-administrators
     ```
 
-    **Role ARN**: should be one of the following depending on the level of access you have been configured with [here](https://github.com/alphagov/govuk-aws-data/blob/master/data/infra-security/integration/common.tfvars):
-
-        * govuk-admins: arn:aws:iam::210287912431:role/govuk-administrators
-        * govuk-poweruser: arn:aws:iam::210287912431:role/govuk-poweruser
-        * govuk-users: arn:aws:iam::210287912431:role/govuk-users
+  * govuk-poweruser:
 
-    **MFA ARN**: This is the long string next to `Assigned MFA device` in the AWS console
+    ```
+    arn:aws:iam::210287912431:role/govuk-poweruser
```

Comment:
Please can we not put full ARNs in this repo as it is public. These must have slipped through the net in the last page update.